.github/CODEOWNERS

```diff
@@ -49,7 +49,8 @@ src/lib/core/datetime/**           @mmalerba
 # CDK
 src/cdk/coercion/**                @jelbourn
 src/cdk/rxjs/**                    @jelbourn
-src/cdk/observe-content/**         @crisbeto @jelbourn
+src/cdk/observe-content/**         @jelbourn @crisbeto
+src/cdk/collections/**             @jelbourn @andrewseguin
```

Comment:
Can you add me to this one as well since I did the `SelectionModel` initially? Also if we're making the `SelectionModel` a part of the CDK, should we remove the `@docs-private`?